internal/service/alertconfiguration/resource_alert_configuration_test.go

```diff
@@ -381,10 +404,21 @@ func TestAccConfigRSAlertConfiguration_withDataDog(t *testing.T) {
 		CheckDestroy:             checkDestroyUsingProxy(proxyPort),
 		Steps: []resource.TestStep{
 			{
-				Config: configWithDataDog(projectID, ddAPIKey, ddRegion, true),
+				Config: configWithDataDog(projectID, ddAPIKey, ddRegion, true, ddNotificationMap, groupNotificationMap),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					checkExistsUsingProxy(proxyPort, resourceName),
+					resource.TestCheckResourceAttr(resourceName, "project_id", projectID),
+					resource.TestCheckTypeSetElemNestedAttrs(resourceName, "notification.*", groupNotificationMap),
```

Comment:
similar/same checks in different tests can be extracted to a helper check method